Converters/ArcGIS/Speckle.Converters.ArcGIS3/Utils/ArcGISFieldUtils.cs

```diff
@@ -186,49 +186,10 @@ List<string> fieldAdded
   {
     if (field.Value is Base attributeBase)
     {
-      // only traverse Base if it's Rhino userStrings, or Revit parameter, or Base containing Revit parameters
-      if (field.Key == "parameters")
-      {
-        foreach (KeyValuePair<string, object?> attributField in attributeBase.GetMembers(DynamicBaseMemberType.Dynamic))
-        {
-          // only iterate through elements if they are actually Revit Parameters or parameter IDs
-          if (
-            attributField.Value is Objects.BuiltElements.Revit.Parameter
-            || attributField.Key == "applicationId"
-            || attributField.Key == "id"
-          )
-          {
-            KeyValuePair<string, object?> newAttributField =
-              new($"{field.Key}.{attributField.Key}", attributField.Value);
-            Func<Base, object?> functionAdded = x => (function(x) as Base)?[attributField.Key];
-            TraverseAttributes(newAttributField, functionAdded, fieldsAndFunctions, fieldAdded);
-          }
-        }
-      }
-      else if (field.Key == "userStrings")
-      {
-        foreach (KeyValuePair<string, object?> attributField in attributeBase.GetMembers(DynamicBaseMemberType.Dynamic))
-        {
-          KeyValuePair<string, object?> newAttributField = new($"{field.Key}.{attributField.Key}", attributField.Value);
-          Func<Base, object?> functionAdded = x => (function(x) as Base)?[attributField.Key];
-          TraverseAttributes(newAttributField, functionAdded, fieldsAndFunctions, fieldAdded);
-        }
-      }
-      else if (field.Value is Objects.BuiltElements.Revit.Parameter)
-      {
-        foreach (
-          KeyValuePair<string, object?> attributField in attributeBase.GetMembers(DynamicBaseMemberType.Instance)
-        )
-        {
-          KeyValuePair<string, object?> newAttributField = new($"{field.Key}.{attributField.Key}", attributField.Value);
-          Func<Base, object?> functionAdded = x => (function(x) as Base)?[attributField.Key];
-          TraverseAttributes(newAttributField, functionAdded, fieldsAndFunctions, fieldAdded);
-        }
-      }
-      else
-      {
-        // for now, ignore all other properties of Base type
-      }
+      // Revit parameters are sent under the `properties` field as a `Dictionary<string,object?>`.
+      // This is the same for attributes from other applications. No Speckle objects should have attributes of type `Base`.
+      // Currently we are not sending any rhino user strings.
+      // TODO: add support for attributes of type `Dictionary<string,object?>`
```

Comment:
we can just move this part of code lower to the condition "if Dict" rather than if Base. Why removing if the TODO is already done here?

Reply:
Let's sync on this: archicad, civil3d, revit, and eventually tekla will send some properties/parameters under the `properties` field with a dict value - need to implement this in a way in arcgis that can handle diff levels of nesting